components/filemanager/overview.md

```diff
@@ -0,0 +1,423 @@
+---
+title: Overview
+page_title: FileManager Overview
+description: Overview of the FileManager for Blazor.
+slug: filemanager-overview
+tags: telerik,blazor,filemanager,overview
+published: True
+position: 0
+---
+
+# Blazor FileManager Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/filemanager" target="_blank">Blazor FileManager component</a> is an Explorer-like component enabling you to manage file and folders.
+
+
+## Creating Blazor FileManager
```

Comment:
The three steps here are simple, while the example itself is complex. This makes the steps unnecessary, as they cannot guide the developer to configure the component. Can we make the example simpler, while the instructions more complex, so they meet half-way? For example - 

* `OnModelInit` can be present in the steps, if it's required.
* `OnDownload` can be removed, if it doesn't work anyway.
* Devote one bullet for `@bind-Path`.
* Devote one bullet for each event - for example, describe what does `OnCreate` do.
* `OnCreate` contains a hard-coded Id. What happens if the reader creates something multiple times?
* Ultimately, we can make the basic example "read-only", so that it just lists files.